demo/custom-operator/Rot13Operator.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Vendor\Rx\Operator;
+
+use Rx\ObservableInterface;
+use Rx\Observer\CallbackObserver;
+use Rx\ObserverInterface;
+use Rx\Operator\OperatorInterface;
+use Rx\SchedulerInterface;
+
+class Rot13Operator implements OperatorInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function __invoke(
+        ObservableInterface $observable,
+        ObserverInterface $observer,
+        SchedulerInterface $scheduler = null
+    ) {
+        return $observable->subscribe(new CallbackObserver(
+            function ($json) use ($observer) {
+                $observer->onNext(str_rot13($json));
+            }
```

Comment:
This example doesn't complete, so the example fails when testing.  Can you add `onError` and `onCompleted` handlers to the example?
```PHP
new CallbackObserver(
     function ($json) use ($observer) {
        $observer->onNext(str_rot13($json));
    },
    [$observer, 'onError'],
    [$observer, 'onCompleted']
)
```